src/exchangeRateRouter.ts

```diff
@@ -53,19 +63,15 @@ const isIsoPair = (pair: string): boolean =>
 
 // *** MIDDLEWARE ***
 
-const v1ExchangeRateIsoAdder = (req: any, res: any, next: Function): void => {
+const v1ExchangeRateIsoAdder = (req: Req, res, next): void => {
```

Comment:
Because we're going to use a "extended request" cleaner as a solution to our typing woes, we can now do `: express.RequestRouter`